test/dynamicForm.test.ts

```diff
@@ -128,6 +129,12 @@ describe('Dynamic form component', function () {
         });
     });
 
+    it("emits confirmEditing event when triggered", async () => {
+        const rendered = getWrapper(validFormMeta, {}, mount);
+        rendered.vm.$emit("confirmEditing")
```

Comment:
I may be wrong, but should the emit not come from the DynamicFormControlSection and then you test whether DynamicForm emits in turn?

Reply:
I agree,  testing has been done from DynamicFormControlSection. However, testing from Dynamic-form's parents will be done via hint as there are no implementation in this module for confirmEditing emit as I only emit custom event.